script/spec-runner.js

```diff
@@ -13,6 +13,15 @@ const NPM_CMD = process.platform === 'win32' ? 'npm.cmd' : 'npm'
 
 const specHashPath = path.resolve(__dirname, '../spec/.hash')
 
+let only = null
+const onlyArg = process.argv.find(arg => arg.startsWith('--only='))
```

Comment:
This allows you to do `npm run test -- --only=remote` locally or `npm run test -- --only=main`

Reply:
can we please use yargs :|

do we foresee having more runners than {main, remote}? if not, i'd argue this should be `--only-remote` / `--only-main`. nitpicky tho.

Reply:
> do we foresee having more runners than {main, remote}?

In theory, yes, UI regression tests, app-like tests, etc.

I built it out with the intention of supporting N runners